src/store/modules/game.js

```diff
@@ -100,6 +101,9 @@ export default {
       }
       return state.players[(state.myPNum + 1) % 2];
     },
+    spectatingUsers(state) {
+      return state.spectatingUsers;
```

Comment:
Do you need to be type safe here? 
```
return state.spectatingUsers || [];
```

Reply:
I think we are okay here, because spectatingUsers is always an array